### PR TITLE
7 adjust quantity item in cart

### PR DESCRIPTION
**Pull request by meyerhoferc**
all tests pass
* user can edit quantity of items in cart, sets a minimum of quantity of 1
* adds subtotal column for an item (model in cart class)
* model tests for total & subtotal updated

Needs fixing:
the form for editing quantity needs styling, it's a big large

### CODE REVIEW

File: app/views/carts/show.html.erb [hunk 1]
```diff
@@ -15,7 +17,16 @@
     <tr>
       <td><%= Item.find(item).title %></td>
       <td><%= quantity %></td>
+      <td>
+        <%= form_for(:session, url: cart_path, method: :put) do |f| %>
```
Comment (cmacaulay) [inline app/views/carts/show.html.erb:21]: 👍 Amazing!

File: spec/features/cart/user_can_change_quantity_in_cart_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      expect(page).to have_content("Total: $33.0")
+    end
+
+    fill_in("session[quantity]", with: 2)
+    click_on "Update Cart"
+
+    expect(current_path).to eq('/cart')
+
+    within("table") do
+      expect(page).to have_content("2")
```
Comment (cmacaulay) [inline spec/features/cart/user_can_change_quantity_in_cart_spec.rb:46]: Great looking test 🔥

**Review by cmacaulay — APPROVED** [review]
Looks great! WOOO

File: spec/features/cart/user_can_change_quantity_in_cart_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe "visitor can adjust items in cart" do
```
Comment (CjMoore) [inline spec/features/cart/user_can_change_quantity_in_cart_spec.rb:3]: This is a great test, super thorough.

**Pull request merged**